src/sapling/sapling_validation.cpp

```diff
@@ -56,7 +56,7 @@ bool CheckTransactionWithoutProofVerification(const CTransaction& tx, CValidatio
     }
 
     // Check for overflow valueBalance
-    if (tx.sapData->valueBalance > consensus.nMaxMoneyOut || tx.sapData->valueBalance < -consensus.nMaxMoneyOut) {
+    if (tx.sapData->valueBalance > MAX_MONEY || tx.sapData->valueBalance < -consensus.nMaxMoneyOut) {
```

Comment:
missed the other one :)
```suggestion
    if (tx.sapData->valueBalance > MAX_MONEY || tx.sapData->valueBalance < - MAX_MONEY) {
```

Reply:
pushed.